eng/Versions.props

```diff
@@ -21,20 +21,20 @@
   -->
   <PropertyGroup>
     <SystemCommandLineVersion>2.0.0-beta4.24102.1</SystemCommandLineVersion>
-    <SystemCompositionVersion>7.0.0</SystemCompositionVersion>
-    <SystemIOPipelinesVersion>7.0.0</SystemIOPipelinesVersion>
+    <SystemCompositionVersion>8.0.0</SystemCompositionVersion>
+    <SystemIOPipelinesVersion>8.0.0</SystemIOPipelinesVersion>
     <SystemCollectionsImmutableVersion>8.0.0</SystemCollectionsImmutableVersion>
     <SystemReflectionMetadataVersion>8.0.0</SystemReflectionMetadataVersion>
     <!-- Note: When updating SystemTextJsonVersion ensure that the version is no higher than what is used by MSBuild. -->
-    <SystemTextJsonVersion>7.0.3</SystemTextJsonVersion>
-    <MicrosoftBclAsyncInterfacesVersion>7.0.0</MicrosoftBclAsyncInterfacesVersion>
+    <SystemTextJsonVersion>8.0.0</SystemTextJsonVersion>
+    <MicrosoftBclAsyncInterfacesVersion>8.0.0</MicrosoftBclAsyncInterfacesVersion>
     <MicrosoftCodeAnalysisAnalyzersVersion>3.3.4</MicrosoftCodeAnalysisAnalyzersVersion>
     <MicrosoftCodeAnalysisAnalyzerUtilitiesVersion>3.3.0</MicrosoftCodeAnalysisAnalyzerUtilitiesVersion>
     <MicrosoftCodeAnalysisNetAnalyzersVersion>8.0.0-preview.23468.1</MicrosoftCodeAnalysisNetAnalyzersVersion>
     <MicrosoftDiaSymReaderVersion>2.0.0</MicrosoftDiaSymReaderVersion>
     <MicrosoftExtensionsDependencyInjectionVersion>7.0.0</MicrosoftExtensionsDependencyInjectionVersion>
     <MicrosoftExtensionsLoggingVersion>7.0.0</MicrosoftExtensionsLoggingVersion>
```

Comment:
You'll want to update the rest of the Microsoft.Extensions.Logging* stuff too.

Reply:
And probably https://github.com/dotnet/roslyn/blob/bfcd3e20b5e726a9b1fc7220734e05be2450bb13/eng/SourceBuildPrebuiltBaseline.xml#L30 too.